tests/components/split_to_columns_panel.test.ts

```diff
@@ -0,0 +1,137 @@
+import { Model, Spreadsheet } from "../../src";
+import { setCellContent, setSelection } from "../test_helpers/commands_helpers";
+import {
+  click,
+  setCheckboxValueAndTrigger,
+  setInputValueAndTrigger,
+} from "../test_helpers/dom_helper";
+import { mountSpreadsheet, nextTick, setGrid, spyModelDispatch } from "../test_helpers/helpers";
+
+describe("split to columns sidePanel component", () => {
+  let model: Model;
+  let fixture: HTMLElement;
+  let parent: Spreadsheet;
+  let dispatch: jest.SpyInstance;
+  let confirmButton: HTMLButtonElement;
+  let checkBox: HTMLInputElement;
+
+  beforeEach(async () => {
+    ({ parent, model, fixture } = await mountSpreadsheet());
```

Comment:
As said verbally, we could start testing the 'sub-'side panles (not the container as standalone and not test the whole spreadsheet unless required ^^

Reply:
Won 0.9s :partying_face: :tada: